apps/web-portal-client/lib/features/auth/hooks/auth.jsx

```diff
@@ -1,7 +1,8 @@
 import {useContext} from 'react'
-import {useDispatch} from 'react-redux'
-import {error, logging, login} from '../slices'
+import {useDispatch, useSelector} from 'react-redux'
+import {error, logging, login, logout} from '../slices'
 import {AppContext} from '../../../context'
+import {selectAuthError, selectAuthToken, selectLoginState} from '../selectors'
 
 export function useAuth() {
   const dispatch = useDispatch()
```

Comment:
Looks like we have to wrap `dispatch` with `useCallback`

https://react-redux.js.org/next/api/hooks#examples